org.eclipse.lsp4j.jsonrpc/src/main/java/org/eclipse/lsp4j/jsonrpc/RemoteEndpoint.java

```diff
@@ -47,8 +49,13 @@
 		} else {
 			LOG.log(Level.SEVERE, "Internal error: " + throwable.getMessage(), throwable);
 			ResponseError error = new ResponseError();
-			error.setMessage("Internal error, please look at the server's logs.");
+			error.setMessage("Internal error.");
```

Comment:
probably `throwable.getMessage()` should be also part of a message

Reply:
The first line of the stack trace includes the message.